kernel/dev/rtc/rtc.cpp

```diff
@@ -62,7 +62,7 @@ uint8_t px_rtc_get_register(int reg) {
 void px_read_rtc() {
     // Previous values from RTC
     // Used as a cache to check if we should update
-    uint8_t century, last_second, last_minute, 
+    uint8_t px_rtc_century, last_second, last_minute, 
```

Comment:
Yes. It said it could be uninitialized, so I decided it's best to just let it get its value since we hard coded the check anyways. RTC still gives the right date, so ¯\_(ツ)_/¯